### PR TITLE
Update ui_comp_template_literals.md

### PR DESCRIPTION
**Pull request by ryantfowler**
I found two links that linked to M2.1, whereas this is a M2.2 page.

### CODE REVIEW

File: guides/v2.1/ui_comp_guide/concepts/ui_comp_template_literals.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Template literals allow UI Components to easily assign dynamic values to class p
 
 ### The `defaults` Class Property
 
-UI Components are [associated with Javascript classes](http://devdocs.magento.com/guides/v2.1/ui_comp_guide/concepts/ui_comp_uiclass_concept.html) to handle behavior on the client side. These should extend one of the core classes to provide a base level of functionality. Inside the child class, a `defaults` property can be provided.
+UI Components are [associated with Javascript classes](http://devdocs.magento.com/guides/v2.2/ui_comp_guide/concepts/ui_comp_uiclass_concept.html) to handle behavior on the client side. These should extend one of the core classes to provide a base level of functionality. Inside the child class, a `defaults` property can be provided.
```
Comment (jeff-matthews) [inline guides/v2.1/ui_comp_guide/concepts/ui_comp_template_literals.md]: @ryantfowler, why are you linking to 2.2 from a 2.1 topic when the 2.1 topic is legitimate?

Comment (ryantfowler) [reply guides/v2.1/ui_comp_guide/concepts/ui_comp_template_literals.md]: @jeff-matthews hi, I apologize if I have made a mistake :(.

**Comment by ryantfowler** [comment]
@jeff-matthews no, definitely not. We're good, thank you :). I'm still getting a handle on the idiosyncrasies, so thanks for assisting me with that.

I'll take the suggested action.

Thank you!

**Comment by ryantfowler** [comment]
@jeff-matthews I'm closing this in favor of PR [#1878](https://github.com/magento/devdocs/pull/1878)

**Pull request closed without merging**